gtdynamics/utils/Initializer.h

```diff
@@ -0,0 +1,223 @@
+/* ----------------------------------------------------------------------------
```

Comment:
nit, would have been better if this was renamed and adapted from initialize_solutions.h, for reviewing/history.